service/service.go

```diff
@@ -32,7 +32,7 @@ type settings struct {
 	Factories component.Factories
 
 	// StartInfo provides application start information.
-	StartInfo component.ApplicationStartInfo
+	StartInfo component.BinaryInfo
```

Comment:
```suggestion
	BinaryInfo component.BinaryInfo
```

Same thing here, carry this through to any place using `StartInfo` or `startInfo`.